paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButtonContainerFragment.kt

```diff
@@ -109,6 +108,7 @@ internal class PaymentSheetPrimaryButtonContainerFragment : BasePrimaryButtonCon
             getString(R.string.stripe_setup_button_label)
         }
 
+        viewBinding.primaryButton.isEnabled = viewModel.selection.value != null
```

Comment:
This might feel like a step backwards, but we will be able to remove these one-off value accesses in https://github.com/stripe/stripe-android/pull/6260.